.ci/azure-pipelines-aws-agent.yml

```diff
@@ -30,9 +30,9 @@ jobs:
           git clone --quiet --depth 1 --single-branch https://github.com/apple/swift-corelibs-libdispatch swift-corelibs-libdispatch
         displayName: 'Fetch Sources'
       - powershell: |
-          (New-Object System.Net.WebClient).DownloadFile("http://download.icu-project.org/files/icu4c/63.1/icu4c-63_1-Win64-MSVC2017.zip", "$(Build.StagingDirectory)\\icu4c-63_1-Win64-MSVC2017.zip")
+          (New-Object System.Net.WebClient).DownloadFile("http://download.icu-project.org/files/icu4c/64.2/icu4c-64_2-Win64-MSVC2017.zip", "$(Build.StagingDirectory)\\icu4c-64_2-Win64-MSVC2017.zip")
```

Comment:
I think that it is probably better to avoid this now entirely.  We have locally built ICU, lets use the local build.